presto-main/src/main/java/com/facebook/presto/memory/QueryContext.java

```diff
@@ -91,7 +99,8 @@ public QueryContext(
             Executor notificationExecutor,
             ScheduledExecutorService yieldExecutor,
             DataSize maxSpill,
-            SpillSpaceTracker spillSpaceTracker)
+            SpillSpaceTracker spillSpaceTracker,
+            DataSize maxBroadcastUsedMemory)
```

Comment:
nit: move this argument up by maxUserMemory and maxTotalMemory (doing refactor -> changeSignature in intellij can reorder the arguments for you)